bedrock/firefox/helpers.py

```diff
@@ -145,14 +152,15 @@ def download_firefox(ctx, channel='release', small=False, icon=True,
     data = {
         'locale_name': locale_name,
         'version': version,
-        'product': 'firefox-android' if platform == 'android' else 'firefox',
+        'product': 'firefox-%s' % platform,
```

Comment:
Must be. As far as I can tell this change seems fine and has no affect. Obviously this changes the value since it can no longer be just "firefox", but it doesn't seem to be used.